addons/account/models/account_journal.py

```diff
@@ -353,7 +353,7 @@ def _constrains_account_control_ids(self):
             WHERE aml.journal_id in (%s)
             AND EXISTS (SELECT 1 FROM journal_account_control_rel rel WHERE rel.journal_id = aml.journal_id)
             AND NOT EXISTS (SELECT 1 FROM journal_account_control_rel rel WHERE rel.account_id = aml.account_id AND rel.journal_id = aml.journal_id)
-            AND aml.display_type IS NULL
+            AND (aml.line_type IS NULL OR aml.line_type NOT IN ('invl_section', 'invl_note'))
```

Comment:
`aml.line_type NOT IN ('invl_section', 'invl_note')` is enough

Reply:
Are you sure? Without it the test that tests this break.

If I try in psql "select :val NOT IN ('invl_section', 'invl_note')" and set val to null, it returns <null> and not True

Reply:
`COALESCE(aml.line_type NOT IN ('invl_section', 'invl_note'), FALSE)` ?